src/Components/Search.tsx

```diff
@@ -47,9 +47,10 @@ const Search = () => {
         }
       }
       
-      const debounceFn = myDebounce((value1: string) => {
-        const suggestions = searchSuggestions(value1)
-        setSugg(suggestions?.slice(0, 10))
+      const debounceFn = myDebounce(async (value1: string) => {
```

Comment:
can you tell me why you had to use async await here?

Reply:
Here I called a async function. That's why I used async await.